react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -2,20 +2,23 @@ import { useState } from 'react';
 import './App.css';
 
 export function ToggleSwitch() {
-  const [toggle, setToggle] = useState('off');
-
+  const [toggle, setToggle] = useState(true);
```

Comment:
The problem with the word "toggle" is that it's not clear if `toggle === true` means the switch is on or off. You should try to find a name that clearly indicates which it is. My preference is `isOn`, but there are other choices that work.